### PR TITLE
Add input pattern config

### PR DESCRIPTION
**Pull request by philipp94831**

### CODE REVIEW

File: charts/streams-app-cleanup-job/templates/job.yaml [hunk 1]
```diff
@@ -92,6 +96,10 @@ spec:
             - name: "{{ .Values.configurationEnvPrefix }}_EXTRA_INPUT_TOPICS"
               value: "{{- range $key, $value := .Values.streams.extraInputTopics }}{{ $key }}={{ $value }},{{- end }}"
             {{- end }}
+            {{- if and (hasKey .Values.streams "extraInputPatterns") (.Values.streams.extraInputPatterns) }}
+            - name: "{{ .Values.configurationEnvPrefix }}_EXTRA_INPUT_PATTERNS"
+              value: "{{- range $key, $value := .Values.streams.extraInputPatterns }}{{ $key }}={{ $value }},{{- end }}"
```
Comment (yannick-roeder) [inline charts/streams-app-cleanup-job/templates/job.yaml:101]: The ending `,` does not hurt us here?

Comment (philipp94831) [reply charts/streams-app-cleanup-job/templates/job.yaml:101]: No (see extra inputs above)

File: azure-pipelines.yml [hunk 2]
```diff
@@ -18,6 +18,7 @@ resources:
     - repository: templates
       type: github
       name: bakdata/bakdata-project-templates
+      ref: tmp/upload-snapshot
```
[the remark this answers is not in the document]
Comment (philipp94831) [reply azure-pipelines.yml]: thanks, done

**Review by yannick-roeder — APPROVED** [review]

**Review by VictorKuenstler — COMMENTED** [review]
LGTM, but maybe we should mention in the description that the error topic does not have a default topic name anymore?

**Comment by philipp94831** [comment]
> LGTM, but maybe we should mention in the description that the error topic does not have a default topic name anymore?

Oh wow, there were docs. Updated them @VictorKuenstler

**Review by VictorKuenstler — APPROVED** [review]

**Pull request merged**